Python/Pomodoro_Timer/Pomodoro.py

```diff
@@ -0,0 +1,152 @@
+import time
+
+# path of host file in windows
+host_path = r"C:\Windows\System32\drivers\etc\hosts"
+
+# URL of websites to block
```

Comment:
Is there a way, to possibly provide these URLs as a `txt`, `csv`, or a `json` file?